src/templates/cms/includes/sidebar.template.html

```diff
@@ -1,145 +1,48 @@
 <aside class="col-12 col-md-3" id="left-sidebar">
 	[%IF [@page_type@] eq 'category' and [@config:show_filters@]%]
+		[%site_value id:'footer_javascript'%]
+			<script>
+			$(document).ready(function() {
+				var seperator = window.location.href.indexOf("?") > -1 ? "&" : "?";
+				var url = window.location.href + seperator + "templatebody=filters&templatehead=none&templatefoot=none"
+
+				$.get(url, function(data) {
+					$('#filters').html(data);
+
+					// Init price slider
+					var dataset = document.querySelector('#price-filter').dataset
```

Comment:
Browser support for `dataset` is also questionable... 

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset

Reply:
IE10 is disabled by default for TSL 1.2 (whatever that means?)
![screen shot 2018-03-20 at 11 40 01 am 1](https://user-images.githubusercontent.com/11847839/43619681-63cd46c2-9712-11e8-801f-3a754871542a.png)